labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -33,8 +34,17 @@ object CollectionExercise01 {
    *
    */
   def googleCodeJamGooglerese(lines: String*): Seq[String] = {
-    error("fix me")
+    val input = "ejp mysljylc kd kxveddknmc re jsicpdrysi rbcpc ypc rtcsra dkh wyfrepkym veddknkmkrkcd de kr kd eoya kw aej tysr re ujdr lkgc jv"
+    val output = "our language is impossible to understand there are twenty six factorial possibilities so it is okay if you want to just give up"
+    var letterMap = stringsToMap(input, output)    
+    
+    lines.map(str => str.map(c => letterMap(c)))
```

Comment:
Just a heads up, I believe you're looking at a wrong pull request, since this is from DSL last year. You may also be sending email notification to a bunch of people who are no longer in DSL or at Mudd.